autograder/cli/courses/users/enroll-file.py

```diff
@@ -0,0 +1,98 @@
+import sys
+
+import autograder.api.constants
+import autograder.api.courses.users.enroll
+import autograder.cli.common
+import autograder.util.hash
+
+def run(arguments):
+    arguments = vars(arguments)
+
+    arguments['raw-course-users'] = _load_users(arguments['path'])
+    arguments['send-emails'] = not arguments['skip-emails']
+
+    result = autograder.api.courses.users.enroll.send(arguments, exit_on_error = True)
+
+    autograder.cli.common.list_user_op_responses(result['results'], table = arguments['table'])
+    return 0
+
+def _load_users(path):
```

Comment:
I don't want you to put too much work into it (since it will only be a small gain),
but see if there is code to be shared with autograder/cli/users/upsert-file.py.
Maybe there can be some common TSV reading we put in util and then this function goes through a [][]string (using Go jargon in this comment) for all of it's final validation/parsing.